packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.tsx

```diff
@@ -247,7 +247,9 @@ export class ContextualMenu extends BaseComponent<IContextualMenuProps, IContext
     let contextMenuStyle;
     let targetAsHtmlElement = this._target as HTMLElement;
     if ((useTargetWidth || useTargetAsMinWidth) && targetAsHtmlElement && targetAsHtmlElement.offsetWidth) {
```

Comment:
you're checking offsetWidth, but calling getBoundingClientRect, while it's probably fine we should still change it to be consistent.

Reply:
What's the appropriate check here then? I would've thought checking `offsetWidth` initially was unnecessary anyway.